api/migrations/0002_auto_20150430_1743.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='formdata',
+            name='form_id',
+            field=models.CharField(max_length=255, verbose_name='Form CSS selector', blank=True),
+        ),
+        migrations.AlterField(
+            model_name='formdata',
+            name='css_classes',
+            field=models.TextField(verbose_name='Form CSS classes'),
+        ),
+        migrations.AlterField(
+            model_name='formdata',
+            name='elements_css_classes',
+            field=models.TextField(verbose_name='Field CSS classes'),
+        ),
+        migrations.AlterField(
+            model_name='formdata',
+            name='html',
+            field=models.TextField(verbose_name='HTML', editable=False),
+        ),
+        migrations.AlterField(
+            model_name='formdata',
+            name='method',
+            field=models.CharField(max_length=4, default='POST', verbose_name='Form method', choices=[('GET', 'GET'), ('POST', 'POST')]),
+        ),
+    ]
```

Comment:
What will developers do that have the `develop` branch already checked out and applied the previous migration(s)? They have to delete their database because there is no migration path for them.

But it could make sense to squash all migrations created since the last release before `develop` is merged into `master`.